spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -33,11 +33,11 @@ To meet those requirements, we introduce 2 new claims to the MP-JWT:
 
 * "upn": A human readable claim that uniquely identifies the subject or user principal of the token, across
 the MicroProfile services the token will be accessed with.
-* "groups": The token subject's group memberships that will be mapped to Java EE style application
-level roles in the MicroProfile service container.
+* "groups": A list of group names that have been assigned to the principal of the MP-JWT. This typically will require a mapping at the application container level to application deployment roles, but a one-to-one between group names and application role names is required to be performed in addition to any other mapping when this claim is available in the token.
```

Comment:
Maybe change `that have been assigned` to `to be assigned`?

Change `application deployment roles` to just `application roles`.

Reply:
@radcortez This is the original spec text. I'm assuming `have been assigned` means that these groups have already been allocated to a given client/user somewhere in the OIDC admin console (ex Keycloak) and next - they may need to be mapped to the deployment specific roles.
 
`application deployment roles` -> `application roles` - fine with me

Reply:
I interpret that as the token containing the `groups` and being used to produce the `Principal`. Because `groups` already exist in the token before the `Principal` I think it would be more clear as a future action `to be assigned`.

I guess it is only semantics :) I'm fine either way.

Reply:
Sure, I've no problems really too, so will update